### DISCUSSION TITLE
Multi-Subscripts seem to exit KaTeX altogether

### DISCUSSION THREAD

**Discussion by DasNaCl** [KaTeX/KaTeX#2818] [Q&A] [answered]
This is a weird one. I'm trying to render `f(e^A_i) = a_{1,i} e^B_1 + \cdots + a_{m,i} e^B_m`, which works just fine on the katex.org web-widget.
If I embed this on my `hugo`-based site, however, things derail quickly. The equation is not rendered and no errors are shown.

It seems like whenever I use something along the lines of `a_{42}` KaTeX just does nothing, no error and no rendering. Purposefully adding an error here also doesn't emit an error, however, it does emit one when I remove the `a_{42}`....

Not sure how I even debug this, since no error is thrown. :frowning_face:

**Comment by DasNaCl** [comment, marked as answer]
I see, escaping using `f(e^A_i) = a\_{1,i} e^B_1 + \cdots + a\_{m,i} e^B_m` makes it work.
It's interesting that I have to escape `_` as soon as it pairs with `{`

Changing the processor from the someday to be deprecated `backfriday` to `goldmark` lets you use above expression without escaping.

**Reply by edemaine** [reply]
Yeah, I imagine it's a Markdown conflict of some sort.  In particular `_foo_` puts `foo` into italics, so multiple `_`s could cause problems in this way, unless you're using a Hugo KaTeX plugin which should properly parse and escape `$...$`.
